packages/ui/src/views/MainContent.vue

```diff
@@ -127,6 +140,15 @@ export default {
       }
     })
 
+    onMounted(() => {
+      playerInfoPoller = setInterval(async () => {
+        await player.getPlayerInfo()
+      }, 5000)
```

Comment:
are we polling the api every 5 seconds? If so, I'd like to move this to an environment variable